Bookish.Web/Controllers/HomeController.cs

```diff
@@ -73,6 +72,11 @@ public IActionResult NewBookPost(string title, string author, string isbn, int t
         [Route("/Home/Barcodes/{isbn}")]
         public IActionResult Barcodes(string isbn)
         {
+            if (!bookishService.DoesIsbnExist(isbn))
+            {
+                return StatusCode(404);
+            }
+
             var model = new BarcodeViewModel(barcodeService.GetNewBookBarcodes(isbn));
```

Comment:
If the isbn doesn't exist then GetNewBookBarcodes should return an empty list right? So just checking that would be more efficient 🙂

Reply:
GetNewBookBarcodes() returns a NewBook object with some book information along with the list of barcodes, so if the book doesn't exist the error needs to be handled before obtaining the title and author. Should the method return null which can then be checked by the controller?

Reply:
Yes, I would expect GetNewBookBarcodes to return `null` if there are no books with that isbn 🙂